services/src/main/java/edu/unc/lib/dl/cdr/services/rest/modify/ExportCsvController.java

```diff
@@ -173,14 +177,48 @@
         return null;
     }
 
+    private String computePatronPermissions(List<String> roles) {
+        if (roles == null) {
+            return "Staff-only";
+        }
+
+        Map<String, String> roleList = new HashMap<>();
+        for (String role : roles) {
+            String[] principalRole = role.split("\\|");
+            roleList.put(principalRole[1], principalRole[0]);
+        }
+
+        String everyoneRole = roleList.get("everyone");
```

Comment:
use constants for this and the next line
https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/fcrepo4/security/src/main/java/edu/unc/lib/dl/acl/util/AccessPrincipalConstants.java#L26